### ISSUE TITLE
Having an issue installing on windows (esoteric error message - npm not finding a file)

### ISSUE THREAD

**Issue by rbkn** [mulesoft/api-workbench#140]
I'm currently getting an esoteric error when attempting to install this via apm on Windows.
Any ideas on what might be causing this?  Is this related to a version conflict for node?

Visual Studio C++ 2013 Express is installed as per install instructions/requirements.

-----<ERROR BELOW>----

>apm install api-workbench
Installing api-workbench to C:\Users\user\.atom\packages failed

npm WARN engine escodegen@1.8.0: wanted: {"node":">=0.12.0"} (current: {"node":"
0.10.40","npm":"2.13.3"})
npm WARN deprecated lodash@1.2.1: lodash@<3.0.0 is no longer maintained. Upgrade
 to lodash@^4.0.0.
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Users\\user\\AppData\\Local\\atom\\app-1.5.4\\resources\\
app\\apm\\bin\\node.exe" "C:\\Users\\user\\AppData\\Local\\atom\\app-1.5.4\\
resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\U
sers\\user\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\knightsr\\.atom\
\.apmrc" "install" "C:\\Users\\user\\AppData\\Local\\Temp\\d-116221-4644-1uf
tsj2\\package.tgz" "--target=0.34.5" "--arch=ia32"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.3
npm ERR! path C:\Users\user\AppData\Local\Temp\apm-install-dir-116221-4644-1
ln22pz\node_modules\api-workbench\node_modules\base64url\node_modules\meow\node_
modules\indent-string\node_modules\repeating\node_modules\is-finite\node_modules
\number-is-nan\package.json.a33229fcee17ad8e48f4dd933e811dea
npm ERR! code ENOENT
npm ERR! errno 34

npm ERR! enoent ENOENT, open 'C:\Users\user\AppData\Local\Temp\apm-install-d
ir-116221-4644-1ln22pz\node_modules\api-workbench\node_modules\base64url\node_mo
dules\meow\node_modules\indent-string\node_modules\repeating\node_modules\is-fin
ite\node_modules\number-is-nan\package.json.a33229fcee17ad8e48f4dd933e811dea'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent

**Comment by thenanox** [comment]
not from mulesoft team, just trying to help:

it seems you have node 0.10 installed, maybe is beacuse of node version? try to upgrade version at least 0.12 or 4.4.0 LTS

hope it helps

**Comment by rbkn** [comment]
Thanks for the comment @thenanox.

Actually I think that version of node is packaged with the win64 version of atom under
C:\Users\user\AppData\Local\atom\app-1.5.4\resources\app\apm\bin

The version I have actually installed (again the windows version - no choice here) is v5.4.0.

**Comment by thenanox** [comment]
yes you are right @rbkn , i checked on there and apm bin uses 0.10 node version to install.

**Comment by sichvoge** [comment]
Just having a quick look and it also seems to me that it is indeed a version conflict. But I am not an expert on this.

**Comment by rbkn** [comment]
Thanks @sichvoge, I'm just not sure how to resolve it - surely the package versions in use should be the one in use in the Atom environment under apm/npm?  Not sure how to modify that...?
